### PR TITLE
Fix privacy notes (DEV)

### PR DESCRIPTION
**Pull request by SamuraiKek**
Addressing the comments from #5906 
Please have another look.

### CODE REVIEW

File: Corona-Warn-App/src/main/assets/privacy_de.html
Comment (MikeMcC399) [inline Corona-Warn-App/src/main/assets/privacy_de.html:1]: In line 1
"Hinweise zur Beendigung der Verarbeitung personenbezogener"

is incomplete. I would expect "Daten" at the end.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Word missing at end of `h1` at top of document.

image

File: Corona-Warn-App/src/main/assets/privacy_de.html
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/assets/privacy_de.html:1]: Thanks, I will update it.

**Review by MikeMcC399 — APPROVED** [review]
Changed heading (DE) looks fine now. 👍🏻

**Review by OLVI5 — APPROVED** [review]

**Pull request merged**